ProjectManager/ProjectManager/Util/UIView+Extension.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UIView+Extension.swift
+//  ProjectManager
+//
+//  Created by Minseong on 2022/07/13.
+//
+
+import UIKit
+
+extension UIView {
+  func shadow(
+    borderWidth: CGFloat,
+    borderColor: CGColor,
+    shadowColor: CGColor,
+    shadowOffset: CGSize,
+    shadowOpacity: Float
+  ) {
+    self.layer.masksToBounds = false
+    self.layer.borderWidth = borderWidth
+    self.layer.borderColor = borderColor
+    self.layer.shadowColor = shadowColor
+    self.layer.shadowOffset = shadowOffset
+    self.layer.shadowOpacity = shadowOpacity
+  }
```

Comment:
모든 파라미터를 다 입력해야 하는 것보다 default value를 지정해줘서 필요한 값만 custom에서 뷰를 설정할 수 있게 하는건 어떨까요?
```suggestion
  func shadow(
    borderWidth: CGFloat = .zero.,
    borderColor: CGColor = .clear,
    shadowColor: CGColor,
    shadowOffset: CGSize,
    shadowOpacity: Float
  ) {
    self.layer.masksToBounds = false
    self.layer.borderWidth = borderWidth
    self.layer.borderColor = borderColor
    self.layer.shadowColor = shadowColor
    self.layer.shadowOffset = shadowOffset
    self.layer.shadowOpacity = shadowOpacity
  }
```

Reply:
훨 좋은 것 같습니다👍